src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -1,38 +1,66 @@
+import PropTypes from 'prop-types';
 import * as React from 'react';
 import {Pressable, StyleSheet, Text, View} from 'react-native';
 import SimpleToast from 'react-native-simple-toast';
-import PropTypes from 'prop-types';
-
-import FastImage from 'react-native-fast-image';
-import TopicMemberIcon from '../../../assets/images/topic-member-picture.png';
+import CommunityIcon from '../../../assets/icon/CommunityIcon';
+import {Shimmer} from '../../../components/Shimmer/Shimmer';
+import {colors} from '../../../utils/colors';
 import {fonts, normalize, normalizeFontSize} from '../../../utils/fonts';
 import {convertString} from '../../../utils/string/StringUtils';
-import {colors} from '../../../utils/colors';
 
 const TopicDomainHeader = (props) => {
   const {domain, handleOnMemberPress, hideSeeMember, isFollow, memberCount} = props;
+  const isUserFollow = props?.initialData?.isFollowing ? props.initialData.isFollowing : isFollow;
+
   const handlePress = () => {
-    if (isFollow) {
+    if (isUserFollow) {
       handleOnMemberPress();
     } else {
       SimpleToast.show('Only community members can see other members', SimpleToast.SHORT);
     }
   };
-
   return (
-    <View>
-      <Text style={styles.domainText} numberOfLines={1} ellipsizeMode="tail">
+    <View
+      style={{
+        alignContent: 'flex-start',
+        alignItems: 'flex-start'
+      }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's better to move inline styles into the StyleSheet object at the bottom of the file for better performance and maintainability.
```diff
- <View
-   style={{
-     alignContent: 'flex-start',
-     alignItems: 'flex-start'
-   }}>
+ <View style={styles.container}>
```
And add this to your styles:
```javascript
container: {
  alignContent: 'flex-start',
  alignItems: 'flex-start'
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->